live-examples/css-examples/transforms/transform-origin.css

```diff
@@ -0,0 +1,60 @@
+@keyframes rotate {
+    from {
+        transform: rotate(0);
+    }
+
+    to {
+        transform: rotate(30deg);
+    }
+}
+
+@keyframes rotate3d {
+    from {
+        transform: rotate3d(0);
+    }
+
+    to {
+        transform: rotate3d(1, 2, 0, 60deg);
+    }
+}
+
+#example-container {
+    width: 160px;
+    height: 160px;
+    position: relative;
+}
+
+#example-element {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    position: absolute;
+    align-items: center;
+    justify-content: center;
+    background: #8edbff;
+    color: rgba(0, 0, 0, .4);
```

Comment:
We are going to add docs for this kinda thing but, I thought I would call it out here. When choosing colours for typography, background colours etc. please refer to the MDN Web Docs front-end style guide(early days, work in progress :)) ~ https://schalkneethling.github.io/mdn-fiori/patterns/variables/

So here, I would suggest using `#29627e` for the background colour, `#fff` for the foreground colour, and `#f6d53c` for the crosshair and border. Thanks @mfluehr